result/convergence.yaml

```diff
@@ -0,0 +1,102 @@
+components:
+  schemas:
+    Convergence.Request:
+      description: >-
+        The container for a CPDP convergence request.
```

Comment:
I think better might be : Container for requesting control-plane and data-plane convergence time metrics for flows. [ Or something that indicates that these are metrics for flows -> Check other metric desc.]